server.js

```diff
@@ -20,7 +35,27 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Hello Claudia')
+})
+
+// endpoint returning 20 thoughts in descending order from created date.
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find().sort({createdAt: 'desc'}).limit(20).exec();
```

Comment:
Lines 43, 44, 45 and 52: Semi colons consistency, in some places I see you use them, but it seems most the code is without them? 👀